tests/OSPSuite.Core.Tests/Core/ModelConstructorIntegrationTests.cs

```diff
@@ -216,16 +216,27 @@ public void Observer_ContainerObs_1_should_have_been_created_under_the_molecule_
          moleculeContainer.ContainsName(observerName).ShouldBeTrue();
       }
 
+
+      [Observation]
+      public void Observer_InContainerObserver_should_have_been_created_under_the_molecule_properties_container_for_C_under_organism()
+      {
+         string observerName = "InContainerObserver";
+
+         var liver_plasma = _model.ModelOrganCompartment(ConstantsForSpecs.Lung, ConstantsForSpecs.Plasma);
```

Comment:
I have created an observer with criteria in Plasma with Parent Lung.... works :)